src/util/runTests.js

```diff
@@ -0,0 +1,49 @@
+import ValidationError from '../ValidationError';
+import { once } from './async';
+
+export default function runTests(options, cb) {
```

Comment:
more big wins here, by making this function less clear with the loop but much shallower